rest-api/src/main/java/com/example/rest_api/entities/Folder.java

```diff
@@ -0,0 +1,30 @@
+package com.example.rest_api.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.Instant;
+import java.util.List;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+@Setter
+@Getter
+public class Folder {
+  @Id
+  @GeneratedValue(strategy = GenerationType.UUID)
+  private String id;
+
+  private String name;
+  private List<Bookmark> bookmarkList;
```

Comment:
why `bookmarkList` instead of just `bookmarks`?

Reply:
yes, it has to `bookmarks`. My bad 😅, I missed to update it before submitting for review.